jetty-websocket/websocket-javax-tests/src/test/java/org/eclipse/jetty/websocket/javax/tests/server/SessionTrackingTest.java

```diff
@@ -104,22 +104,22 @@ public void testAddRemoveSessions() throws Exception
         EventSocket clientSocket2 = new EventSocket();
         EventSocket clientSocket3 = new EventSocket();
 
-        try (Session session1 = client.connectToServer(clientSocket1, server.getWsUri().resolve("/session-info/1")))
+        Session session1 = client.connectToServer(clientSocket1, server.getWsUri().resolve("/session-info/1"));
```

Comment:
I don't understand this change? try-with-resources seems the right thing to do here.
Why did you keep the blocks if you are not using try-with-resources?

Reply:
The close says it should throw `IOException`, so if we have a closed channel it throws `ClosedChannelException`, and because we are already explicitly closing the try-with-resources just always fails.

I can re-structure the code to remove the blocks.

Reply:
But in a double `close()` the second should be a no-op, no? Only in core it throws, but this is javax.

Reply:
The signature javax close signature declares we throw `IOException`.